app/views/pages/home/_signin_jobseekers.html.haml

```diff
@@ -0,0 +1,7 @@
+%h2.govuk-heading-m= t('.title')
+
+%p
+  = t('.description')
+  = link_to t('.signup_link'), new_jobseeker_registration_path, class: 'govuk-link govuk-link--no-visited-state'
```

Comment:
`govuk_link_to`

Reply:
let's do all these changes in another PR

Reply:
Ok, no problem